### PR TITLE
Edits voor user friendlyness

### PR DESCRIPTION
**Pull request by tychografie**
UX upgrades by yours truly, om de OKR's te behalen!

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -57,10 +62,11 @@ export default function Home() {
                 >
                   <Input
                     name="bike"
```
Comment (hiddehs) [inline src/app/page.tsx]: name="name"

File: src/app/page.tsx [hunk 2]
```diff
@@ -57,10 +62,11 @@ export default function Home() {
...
-                    placeholder="The name of your Bike"
+                    placeholder="What is your name?"
                   ></Input>
-                  <Input name="name" placeholder="Your name"></Input>
```
Comment (hiddehs) [inline src/app/page.tsx:62]: name="phone"

**Review by hiddehs — CHANGES_REQUESTED** [review]
amazing contributions 🙏 server route and sql should be changed too
i will look into a db update

**Comment by tychografie** [comment]
hi hidde, i resolved the requested changes.

**Pull request merged**